aspnetcore/mvc/controllers/filters.md

```diff
@@ -199,12 +199,14 @@ If your filters have dependencies that you need to access from DI, there are sev
 
 ### ServiceFilterAttribute
 
-A `ServiceFilter` retrieves an instance of the filter from DI. You add the filter to the container in `ConfigureServices`, and reference it in a `ServiceFilter` attribute
+A `ServiceFilter` retrieves an instance of the filter from DI. Add the `ServiceFilter` to the container in `ConfigureServices`, and reference it in a `ServiceFilter` attribute:
```

Comment:
> add the `ServiceFilter`  

I think is text is misleading. You register the filter **implementation type** in DI. 

Also, should this say `ServiceFilterAttribute` in all of the places it says `ServiceFilter`? See L208